.circleci/config.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           path: htmlcov
   check-translations:
     docker:
-      - image: timoludwig/python-node-gettext:latest
+      - image: integreat/python-node-gettext:78c07dc
```

Comment:
@timoludwig could you please explain why the version for the container is now fixed instead of `latest`?

Reply:
This is done to enable a safe update process.
The images will be pushed to docker hub every time the Dockerfile changes, even on a feature branch. So I added a fixed tag which makes sure that other builds on other branches are still deterministic even if we work on another version of Dockerfiles in another branch. The alternative would have been to only push Dockerfiles when the changes are merged into develop, but then we can't test whether the images work because they are not available on feature branches...
So now, when we want to change something in our Dockerfiles, we

1. Commit the changes to the Dockerfiles
2. Look at the hash of the commit and insert it in the circleci config (and commit this as well)
3. Push the changes and see whether the build is successful on the new feature branch, while all other branches including develop and master still use the old docker images & config.

Reply:
I see. Thanks.